### DISCUSSION TITLE
Episode 11: A Talk with Olivier Mélois

### DISCUSSION THREAD

**Discussion by ckipp01** [ckipp01/tooling-talks#29] [New Speaker Announcement!]
Very excited to announce my next Tooling Talk with Olivier! We'll dive into [Smithy](https://awslabs.github.io/smithy/) and [smithy4s](https://disneystreaming.github.io/smithy4s/) and all the tooling that surrounds it!

![Avatar of Olivier](https://user-images.githubusercontent.com/13974112/168476879-c6e85f94-3423-4cb8-b0ed-7487ddef2d65.jpg)

### Find Olivier Online

@Baccata on GitHub

**Comment by mtk** [comment]
still working on it.  technically, fascinating.  but while olivier's english proficiency is better than 98-99% of *native* speakers, the combo of his accent and some kind of echo made me work really hard to absorb it.  i have no idea how to get clearer sound from a remote interview.  but if you can do something, it would really help.

**Reply by ckipp01** [reply]
Thanks for the feedback @mtk. Unfortunately it's really not something easy on my end to fix. I'm sort of beholden to the setup the other person has on the other side. Do they have an external mic? Are they in a quiet place? Is there room a giant empty room causing echo? etc etc..

With that said, I'm sorry it's hard to understand due to the echo, it's actually just a ton of feedback that somehow got in during the recording on Olivier's side. You only notice a lot when he speaks because I normalized the rest of the audio so it wasn't buzzing the entire time, or you'd hear it even when I was talking.

There are a couple things I've done since this episode though:

1. I came up with some [tips](https://github.com/ckipp01/tooling-talks/blob/main/recording-tips.md) that I can share with guests ahead of time to help make sure it's as clear as possible (although Olivier basically did follow all of these)
2. I've looked into potentially [mailing guests a mic](https://medium.com/@saronyitbarek/mailing-mics-to-podcast-guests-a2f5adf3a906) if they don't have one. Ultimately I'm not at a point where I think I want to do this yet, since I'd be me shilling out money to purchase and ship it all over the world and back, on top of all the effort of just doing the show.

I hope you still enjoyed the episode, as I had a ton of fun recording it and chatting with Olivier :)

**Reply by Baccata** [reply]
The acoustic in the room that I use as an office is unfortunately not great ... good enough for zoom calls, but not great for conference talks and recorded sound. Nothing Chris could have done to make it better, unfortunately... and I didn't want to risk going outside and have the episode polluted with bird noises, horses neighing and the occasional tractor passing by, haha. 

My headset is starting to be a little dated as well. My days of online gaming have been over for a long time, and I prefer spending monies on other things than a pricy microphone that I wouldn't be using for anything serious ... 

Maybe I could have put a mattress against the wall momentarily to make the acoustic less shite, but I didn't realise it was gonna be that bad that people would have a hard time understanding my voice 😅. 

Is there software that can be used to test the acoustics, giving it a score or something ? If that's the case, then future participants could be asked to run it prior to the recording, and see whether they can make small improvements by moving things slightly around them, or changing rooms when they have that luxury, or whatnot ?

**Reply by ckipp01** [reply]
> Is there software that can be used to test the acoustics, giving it a score or something ? If that's the case, then future participants could be asked to run it prior to the recording, and see whether they can make small improvements by moving things slightly around them, or changing rooms when they have that luxury, or whatnot ?

I'll for sure look into this. In some ways [SquadCast](https://squadcast.fm/) has already been a huge help with this with giving me two separate audio files without any flakiness, even with garbage internet at times. The have another echo cancellation feature that I didn't turn on for this episode since it says to turn if *off* if both guests are wearing headphones. I'm curious if that would have helped in this situation.

> and I prefer spending monies on other things than a pricy microphone

I also want to avoid the impression that someone _needs_ this to be on the show. That's originally what caused me to look into shipping a mic around. I don't want anyone to ever say "no" to being on the show because they don't think the audio quality will be good enough. Note to any future person reading this, if that's what is causing you to say no, tell me, and we'll solve it.

There is also probably editing tricks to help with this. While I did normalize everything to ensure that the buzzing wasn't happening the entire time, there are also probably other things I could do. 😅 Just takes time to learn those tricks.

**Reply by Baccata** [reply]
I'd certainly not expect you to ship a microphone over haha, that'd be a huge hassle for both parties. 

I'll personally try to make things better for the next conference talk I give, but otherwise I don't think there's much to be done for the podcast : people have the setup they have and creating a barrier of entry around that would be bad. Unless you have a successful enough Patreon that really helps the podcast, the steps you've already taken to ensure the sound is as good as possible under  circumstances you have little control over are largely enough, imho 😄 

Maybe I should have gone full French in my accent ? Like not pronouncing H, saying "Ze" for "the", haha.

**Comment by mtk** [comment]
still consuming it.  but (and in no way is this an exaggeration) it is a
fascinating episode.  and i'm not just talking about the horses :-).
/mark

**Comment by mtk** [comment]
accents are hard.  your vocabulary puts you *way* beyond fluent (as is
evident in your easily understood written text :-)).  one thing i thought
of after was 'gain' or 'volume'.  chris seemed much louder than you did in
this episode.  in addition to the echo, that also made it difficult to hear
you.  is that a matter of settings?  i have no doubt that face-to-face your
spoken english is completely understandable.

/mark

**Reply by Baccata** [reply]
>  your vocabulary puts you *way* beyond fluent  

hah thanks 😄 

> in addition to the echo, that also made it difficult to hear
you.  is that a matter of settings 

good call. Possibly ?

**Reply by ckipp01** [reply]
> one thing i thought of after was 'gain' or 'volume'.  chris seemed much louder than you did in this episode.  in addition to the echo, that also made it difficult to hear you.  is that a matter of settings?

The actual sounds levels for both of us are almost exactly matched (I needed to do this when I edited it). However because of the loud echo, instead of hearing a nice clear voice, you hear the echo. I could have made it louder, but it would sound funky.

I'm assuming your responses are email? Github discussions doesn't handle email very well.

**Reply by mtk** [reply]
yeah, was responding from email (from the breakfast table :-)).

**Comment by mtk** [comment]
the acoustic software question is something i wondered about too.  but way
outside my expertise.  what did you use to record the conversation?  i
have a friend who has done a music/talk podcast for a decade (called
"fullduplexradio").  he's primarily mac-based (but a deep unix guru too,
one of the implementers of /proc).  i can ask him for suggestions if you
tell me about the recording setup.

/mark

ps - olivier, it wasn't your accent.  you are more fluent that 99% of
*native* english speakers. not joking.

**Reply by ckipp01** [reply]
> what did you use to record the conversation?

I record the talks using https://squadcast.fm/. As I mentioned above, this is primarily just an issue of what mic guests are using and what their environment looks like. I think the episodes after this have had much better audio quality, but again it all depends on the guests setup.

**Reply by Baccata** [reply]
Yeah I was using a 2015 macbook pro on its way to the graveyard and an even older gaming headphones that could do with an upgrade 😓
